core/tests/protractor_desktop/voiceoverUploadFeatures.js

```diff
@@ -148,12 +148,6 @@ describe('Voiceover upload features', function() {
       await explorationEditorTranslationTab.deleteAudioRecord();
     });
 
-  afterAll(async function() {
-    await explorationEditorPage.saveChanges(
-      'Adds audio file in translation tab.');
-    await workflow.publishExploration();
-  });
-
```

Comment:
This seems like a useful part of the test. We do want to make sure we can save and publish changes after adding voiceovers. Instead of removing it, can we figure out why it's causing console errors and fix that?

Reply:
@U8NWXD  As discussed earlier, It's not possible to save changes and publish exploration when there are no users logged in (as users are logged out after each spec). Please let me know if you have any suggestions in mind.

Reply:
Why can't we login in the `afterAll` block?

Reply:
@U8NWXD Do you feel it's appropriate to perform test actions in the afterAll block?